src/index.tsx

```diff
@@ -13,9 +13,13 @@ import {
 import { AppRoute } from "./libs/enum/index.ts";
 import "./libs/locales/i18n.ts";
 import { Auth, Home } from "./pages/index.ts";
+import { PrivacyPolicy } from "./pages/privacy-policy/privacy-policy-page.tsx";
+import { TermsConditions } from "./pages/terms-conditions/terms-and-conditions.tsx";
 import { persistor, store } from "./redux/store.ts";
 import "./styles.css";
 import theme from "./theme.ts";
+import { HeaderWrapper } from "./components/common/wrappers/header-wrapper.tsx";
+import { FooterWrapper } from "./components/common/wrappers/footer-wrapper.tsx";
```

Comment:
```suggestion
import {
	PrivateRoute,
	PublicRoute,
	RouterProvider,
	StoreProvider,
	HeaderWrapper,
	FooterWrapper
} from "./components/common/index.ts";
```
Common components must be exported from the index file

Reply:
even easier
import {
	PrivateRoute,
	PublicRoute,
	RouterProvider,
	StoreProvider,
	HeaderWrapper,
	FooterWrapper
} from "./components/common"
this should work